assets/js/src/containers/App/selectors.js

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from "reselect";
```

Comment:
Reselect is a cool way to access nested state, https://github.com/reactjs/reselect